apps/expo-go/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.kt

```diff
@@ -400,7 +400,7 @@ open class ExperienceActivity : BaseExperienceActivity(), StartReactInstanceDele
       )
       showOrReconfigureManagedAppSplashScreen(optimisticManifest)
       setLoadingProgressStatusIfEnabled()
-      ExperienceRTLManager.setSupportsRTLFromManifest(this, optimisticManifest)
+      ExperienceRTLManager.setRTLPreferencesFromManifest(this, optimisticManifest)
```

Comment:
It may be necessary to implement something similar in the dev client as it also configures the app based on the manifest. Not very important right now, but it would be nice to have that there too.